modules/mssql/outputs.tf

```
@@ -66,3 +66,9 @@ output "root_password" {
   value       = coalesce(var.root_password, random_password.root-password.result)
   sensitive   = true
 }
+
+// Resources
+output "master" {
```

We're trying to stop using `master` in the module.

```suggestion
output "primary" {
```

---

Sorry for the slow response!

I've updated all the references where I could without making the diff too crazy. For example in the `safer_mysql` module the output is now `primary` but it still refers to `module.safer_mysql.master`.